baseapp-chats/baseapp_chats/graphql/mutations.py

```diff
@@ -234,9 +235,23 @@ def mutate_and_get_payload(
         is_leaving_chatroom = profile_id in remove_participants and len(remove_participants) == 1
 
         add_participants_pks = [
-            get_pk_from_relay_id(participant) for participant in add_participants
+            get_pk_from_relay_id(participant)
```

Comment:
If we have to pass relay id's for this, wouldn't this make it really difficult to integrate with our API outside of relay?  (Might be a question that's outside the scope of this PR)

Reply:
Yeah good point. I think we would have to adapt all other functionalities as well. since we are always using relay_id